akka-actor-tests/src/test/scala/akka/pattern/AskSpec.scala

```diff
@@ -1,7 +1,8 @@
 /**
  * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.pattern
+/* Wrapper package has to be added in order to prevent implicits from clashing. */
+package akka.pattern.wrapper
```

Comment:
This file does not add implicits, so this comment seems incorrect; otherwise this file would now also be in the wrong source directory.